.github/workflows/hardening-check.yml

```diff
@@ -0,0 +1,77 @@
+## Copyright 2023-2024 Intel Corporation
+## SPDX-License-Identifier: BSD-3-Clause
+
+name: Run Hardening Checks
+
+permissions: read-all
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
```

Comment:
I think we should run it similar to ClamAV - on release and trunk builds. Otherwise if necessary compiler flag is removed, we'll know about it too late.

Reply:
Done.